### PR TITLE
Temperature 6.1

### PR DESCRIPTION
**Pull request by LukeWalker-CA**
* Update of PCBversion
* Migrated to latest firmware

### CODE REVIEW

File: unit_testing/Temperature/Temperature_tests.cpp [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+#include "fake_ADS1120.h"
+
+/* Real supporting units */
+#include "CAProtocol.c"
+#include "CAProtocolStm.c"
+
+/* UUT */
+#include "Temperature.c"
+
+using ::testing::Contains;
```
Comment (matias-cphatomics) [inline unit_testing/Temperature/Temperature_tests.cpp:26]: I think you can delete line 26-28

**Comment by matias-cphatomics** [comment]
Nice to have unit tests added :1st_place_medal:

**Review by matias-cphatomics — APPROVED** [review]

File: unit_testing/Temperature/Temperature_tests.cpp (hunk 1 above)
Comment (LukeWalker-CA) [reply unit_testing/Temperature/Temperature_tests.cpp:26]: 28 required, removed the other 2

**Comment by LukeWalker-CA** [comment]
Note: ready to merge once it is tested on a board

**Pull request merged**